include/marlin/Configuration_A3ides_2209_MINI.h

```diff
@@ -584,7 +584,7 @@
 // Specify here all the endstop connectors that are connected to any endstop or probe.
 // Almost all printers will be using one per axis. Probes will use one or more of the
 // extra connectors. Leave undefined any used for non-endstop and non-probe purposes.
-#define MINDA_BROKEN_CABLE_DETECTION
+//#define MINDA_BROKEN_CABLE_DETECTION
```

Comment:
do we really want to disable this in master?

Reply:
commit a203014